.vscode/settings.json

```
@@ -18,6 +18,6 @@
   "python.linting.flake8Args": [],
   "[python]": {
     "editor.tabSize": 4,
-    "editor.defaultFormatter": "ms-python.python"
+    "editor.defaultFormatter": "ms-python.black-formatter"
```

```suggestion
    "editor.defaultFormatter": "ms-python.python"
```
こちらも同様です。

pullリクエストによるファイルの差分はgithubのfiles changedタブから見ることができます。
https://github.com/shinonome-inc/backend-final-assignment-Aoto/pull/1/files
こちらがpullリクエストにより変更が加えられる部分を示しているため、「そのpullリクエストにおいて
関係ないところは変えない」感じでお願いします。